virtinst/guest.py

```diff
@@ -235,6 +235,7 @@ def __init__(self, *args, **kwargs):
         self.__osinfo = None
         self._capsinfo = None
         self._domcaps = None
+        self._use_virtio_win = False
```

Comment:
The way this is implemented I think it would be better to name it after what it's specifically doing, so something like force_supports_virtio. exposing that via the --install option too would be a nice addition for users who are manually installing virtio-win drivers. something for later though